app/migrations/seed_dev.py

```diff
@@ -0,0 +1,162 @@
+    if instance:
+        return instance
+    else:
+        instance = TransferAccount(
+            blockchain_address=blockchain_address, organisation=organisation)
+
+        instance.name = name
+        db.session.add(instance)
+        return instance
+
+
+def randomString(stringLength=10):
+    """Generate a random string of fixed length """
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for i in range(stringLength))
+
+
+def create_users_different_transer_usage(wanted_nr_users):
+    i = 1
+    user_list = []
+    transer_usages_ids = TransferUsage.query.with_entities(
+        TransferUsage.id).all()
+    while i < wanted_nr_users:
+        random_usage_id = random.choice(transer_usages_ids)[0]
+        new_user = get_or_create_user(
+            'user-nr-' + str(i) + '@test.com', random_usage_id)
+        new_transer_account = get_or_create_transer_account(
+            'transfer-account-nr-' + str(i), blockchain_address, new_organisation)
+
+        if len(new_user.transfer_accounts) < 1:
+            new_user.transfer_accounts.append(new_transer_account)
+        user_list.append(new_user)
+        i += 1
+    return user_list
+
+
+def create_transfers(sender, user_list, wanted_nr_transfers):
+    transfer_list = []
+    i = 0
+    while i < wanted_nr_transfers:
+        transfer = CreditTransfer(
+                                    i,
+                                    sender_user=user1,
+                                    recipient_user=random.choice(user_list),
+                                    token=token,
+                                    uuid=randomString())
+        db.session.add(transfer)
+        transfer_list.append(transfer)
+        i += 1
+    return transfer_list
+
+
+if __name__ == '__main__':
+
+    app = create_app()
+    ctx = app.app_context()
+    ctx.push()
+
+    print('Creating token')
+    blockchain_address = '0xc1275b7de8af5a38a93548eb8453a498222c4ff2'
+    token = get_or_create_token(blockchain_address,
+                                'BAR Token', 'BAR')
+
+    print('Creating organisation')
+    new_organisation = get_or_create_organisation('org1', token)
+
+    print('Creating admin user')
+    admin_user = get_or_create_admin_user('administrator@sempo.ai', new_organisation)
+
+    print('Creating user 1')
+    user1 = get_or_create_user('sender-user@test.com', 1)
+    transer_account1 = get_or_create_transer_account(
+        'transfer-account', blockchain_address, new_organisation)
+    if len(user1.transfer_accounts) < 1:
+        user1.transfer_accounts.append(transer_account1)
+
+    print('Create a list of users with a different business usage id ')
+    user_list = create_users_different_transer_usage(15)
+
+    print('Creating transactions')
+    # User 1 sends to a random choice of user_list
+    create_transfers(user1, user_list, 30)
+
+    db.session.commit()
+    sql = text("""UPDATE credit_transfer SET transfer_status = 'COMPLETE'""")
```

Comment:
does `credit_transfer.transfer_status = TransferStatusEnum.COMPLETE` (or something to that effect) not work?

Reply:
Yes that works I changed it